src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -64,7 +64,7 @@ export class DynamicFacetManager extends Component {
      */
     enableReorder: ComponentOptions.buildBooleanOption({ defaultValue: true, section: 'Filtering' }),
     /**
-     * A function to execute whenever facets are updated in the query response (see [Implementing Custom Dynamic Facet Behaviors](https://docs.coveo.com/en/2902/#implementing-custom-dynamic-facet-behaviors)).
+     * A function to execute whenever facets are updated in the query response (see [Defining Custom Dynamic Facet Behaviors](https://docs.coveo.com/en/2917/javascript-search-framework/using-dynamic-facets#defining-custom-dynamic-facet-behaviors)).
```

Comment:
Instead of (see ...), you could use an @externaldocs tag:

`@externaldocs [Defining Custom Dynamic Facet Behaviors](https://docs.coveo.com/en/2917/javascript-search-framework/using-dynamic-facets#defining-custom-dynamic-facet-behaviors)`

Similar comment for the other link